scripts/ChangesSinceLastRelease.txt

```diff
@@ -2699,4 +2698,6 @@ Windows.Win32.Graphics.Printing.IPrintTicketProvider.BindPrinter : hPrinter...HA
 Windows.Win32.Graphics.Printing.IPrintTicketProvider.GetSupportedVersions : hPrinter...HANDLE => PRINTER_HANDLE
 Windows.Win32.Graphics.Printing.PRINTER_HANDLE added
 # Remove RAIIFree attribute from PSID
-Windows.Win32.Security.PSID : [NativeTypedef,RAIIFree(FreeSid)] => [NativeTypedef]
\ No newline at end of file
+Windows.Win32.Security.PSID : [NativeTypedef,RAIIFree(FreeSid)] => [NativeTypedef]
+# Correct `CountFieldName` on `DCompositionGetStatistics()`
+Windows.Win32.Graphics.DirectComposition.Apis.DCompositionGetStatistics : targetIds : [Optional,Out] => [NativeArrayInfo(CountFieldName=targetIdCount),Optional,Out]
```

Comment:
Let's undo this and make that change separately. This PR should be solely focused on the revert.

Reply:
Split into #1993